readme.md

```diff
@@ -35,7 +35,7 @@ The NVDA source depends on several other packages to run correctly.
 ### Installed Dependencies
 The following dependencies need to be installed on your system:
 
-* [Python](http://www.python.org/), version 2.7.15, 32 bit
+* [Python](http://www.python.org/), version 3.7.4, 32 bit
```

Comment:
Please consider #8421

Note that on appveyor, we can't ensure that we are on a particular minor release. Therefore, I propose that we advice to always use the most recent minor version

```suggestion
* [Python](http://www.python.org/), most recent minor release of version 3.7, 32 bit
```